README.md

```diff
@@ -1 +1,42 @@
 # bit-docs-prettify
+
+A [bit-docs](https://github.com/bit-docs/bit-docs) plugin that makes source-code snippets in HTML prettier.
+
+Powered by https://github.com/google/code-prettify.
+
+## Quick Start
+
+Add `bit-docs-prettify` as a dependency to your project's `package.json` under the `bit-docs` section:
+
+```json
+{
+  "name": "your-project",
+  ...
+  "bit-docs": {
+    "dependencies": {
+      "bit-docs-prettify": "*"
+    },
+    ...
+  }
+}
+```
+
+Now run `./node_modules/bit-docs/bin/bit-docs` in your project to finish adding `bit-docs-prettify`.
+
+Your `bit-docs` enabled project should now have syntax highlighting of source-code snippets!
```

Comment:
I wasn't actually thinking about languages.  But that's a good call.  The format doesn't matter to me.  

I was thinking of what the HTML needs to look like so this plugin will actually work.  I think it's something like `<pre><code language='js'>`.  Our markdown generator happens to produce this, but it should be stated.